wicker/core/config.py

```diff
@@ -27,12 +27,14 @@ def from_json(cls, data: Dict[str, Any]) -> WickerWandBConfig:
 class WickerAwsS3Config:
     s3_datasets_path: str
     region: str
+    store_concatenated_bytes_files_in_dataset: bool = False
 
     @classmethod
     def from_json(cls, data: Dict[str, Any]) -> WickerAwsS3Config:
         return cls(
             s3_datasets_path=data["s3_datasets_path"],
             region=data["region"],
+            store_concatenated_bytes_files_in_dataset=data.get("store_concatenated_bytes_files_in_dataset", False),
```

Comment:
Nit: Why default to false if you already specify false above, probably redundant.

Reply:
Thank you for your comment!
This is because if you create a `WickerAwsS3Config` object with the `from_json` method, without the default `False` the value of `store_concatenated_bytes_files_in_dataset` will be `None`
 
<img width="1002" alt="スクリーンショット 2023-09-06 8 56 04" src="https://github.com/woven-planet/wicker/assets/91296855/34c1e78a-6f6f-43a0-8325-544d8d164601">